.github/workflows/release.yml

```diff
@@ -167,11 +173,22 @@ jobs:
         with:
           name: tinymist-${{ env.target }}
           path: tinymist-${{ env.target }}${{ fromJSON('["", ".exe"]')[matrix.platform == 'win32'] }}
-      - name: Package extension
+      - name: Package typst-preview extension
+        if: (startsWith(github.ref, 'refs/tags/') || matrix.regular_build == 'true')
```

Comment:
Is it only build when tags are pushed? Is this expected?

Reply:
Either it is tagged or marked as `regular_build` in the matrix (Linux x86, Windows x86, mac ARM)

Reply:
make sense